src/razor/src/razorLanguageServerTraceResolver.ts

```diff
@@ -23,7 +24,7 @@ function parseTraceString(traceString: string | undefined) {
         case 'Verbose':
             return Trace.Verbose;
         default:
-            console.log("Invalid trace setting for Razor language server. Defaulting to 'Off'");
+            console.log(l10n.t("Invalid trace setting for Razor language server. Defaulting to 'Off'"));
```

Comment:
Strictly speaking the `Off` here should not be localized, as it is an accurate representation of what the setting value will be in the UI. And if it's not, then the rest of the switch statement is presumably broken :)

Reply:
I've installed an alternate language and gone through a large swath of settings; the dropdowns in settings, even for core properties, are nearly always untranslated.